Signal/src/Profiles/ProfileFetcherJob.swift

```diff
@@ -105,7 +114,10 @@ class ProfileFetcherJob: NSObject {
     private func updateProfile(signalServiceProfile: SignalServiceProfile) {
         verifyIdentityUpToDateAsync(recipientId: signalServiceProfile.recipientId, latestIdentityKey: signalServiceProfile.identityKey)
 
-        // Eventually we'll want to do more things with new SignalServiceProfile fields here.
+        OWSProfileManager.shared().updateProfile(forRecipientId : signalServiceProfile.recipientId,
```

Comment:
Take it or leave it:

This works, but it's the circular dependency is a little goofy wherein the OWSProfileManager calls the Fetcher Job which then calls back to the OWSProfileManager. Maybe this method should take a block or return a Promise<SignalServiceProfile>

...and then we move the verifyIdentityUpToDate logic into ProfileManager.

All that seems compatible with my earlier mentioned plan of having only *one* block of throttling logic.

This works though.